helm/bankapp/templates/ingress.yml

```
@@ -0,0 +1,20 @@
+spec:
+  rules:
+  - host: bankapp.local # add 
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: "{{ .Values.app_deployment.name }}-service"
+            port:
+              number: {{ default "8080" .Values.bankapp_svc.port}} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)